groups/README.md

```diff
@@ -0,0 +1,33 @@
+# Automation of Google Groups maintenance for `knative.team`
+
+- [Making changes](#making-changes)
+  - [Staging access groups](#staging-access-groups)
+- [Manual deploy](#manual-deploy)
+
+## Making changes
+
+- Edit your SIG's `groups.yaml`, e.g. [`sig-release/groups.yaml`][/groups/sig-release/groups.yaml]
```

Comment:
Should this be updated?

Reply:
Seems like there's quite a few places that are still pointing to k8s land, I reckon they should be updated to local links / our groups?

Reply:
Very early WIP. I need to get access to gsuite and see what is in there and map out the groups and their members